Update has_many.rb (#2274)
thoughtbot/administrate: docs/customizing_dashboards.md

**Field::HasMany**

> `:limit` - Set the number of resources to display in the show view. If not want 
> use paginate in show page, you can set as `0` or `false`. Default is `5`.

Reviewer: ```suggestion
`:limit` - The number of resources (paginated) to display in the show view. To disable pagination,
set this to `0` or `false`. Default is `5`.
```
